src/hotspot/os/bsd/os_perf_bsd.cpp

```diff
@@ -369,6 +369,8 @@ CPUInformationInterface::CPUInformationInterface() {
 }
 
 bool CPUInformationInterface::initialize() {
+  return false;
+#if 0
```

Comment:
The `src/hotspot/os/bsd` path is shared with MacOSX build. We should strive to make the bsd-port branch to be build compatible with MacOSX so that if it was merged into upstream repositories it would not break the build for Apple.
```
#ifndef __APPLE__
  return false;
#else
````
would keep Apple building ok.

Reply:
Looking at the upstream, there is not special Apple treatment for this file, so I'm not sure why we would need to do this.  I think the include fix above is all that is needed here.

Reply:
Ah. Agree.
@MikaelUrankar, now that the header has been fixed does the arm build work with the CPUInformationInterface::initialize() change reverted back?

Reply:
Thanks for fixing my mess :)
I'll try this evening.